src/handler.rs

```diff
@@ -456,7 +456,7 @@ where
         // This request should be handled only when VHOST_USER_F_PROTOCOL_FEATURES
         // has been negotiated.
         if self.acked_features & VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits() == 0 {
-            return Err(VhostUserError::InvalidOperation);
+            return Err(VhostUserError::InvalidOperation("features already set"));
```

Comment:
Isn't it the opposite?  "features not negotiated" or "protocol features not negotiated"

IIUC we enter this branch when we haven't negotiated the features.

Reply:
You're right, I've just updated the string. Thanks!